src/components/App/App.test.js

```diff
@@ -0,0 +1,22 @@
+/* global getRoutesMap:readonly */
```

Comment:
Declaring globals seems to be the most elegant way to share some helper logic in jest. Let me know if you have any other idea for that @blurbyte.